src/data/pokemon/evolution.h

```diff
@@ -47,15 +47,18 @@ const struct Evolution gEvolutionTable[NUM_SPECIES][EVOS_PER_MON] =
     [SPECIES_POLIWHIRL]             = {{EVO_ITEM, ITEM_WATER_STONE, SPECIES_POLIWRATH},
                                        {EVO_TRADE_ITEM, ITEM_KINGS_ROCK, SPECIES_POLITOED}},
     [SPECIES_ABRA]                  = {{EVO_LEVEL, 16, SPECIES_KADABRA}},
-    [SPECIES_KADABRA]               = {{EVO_TRADE, 0, SPECIES_ALAKAZAM}},
+    [SPECIES_KADABRA]               = {{EVO_TRADE, 0, SPECIES_ALAKAZAM},
+                                       {EVO_ITEM, ITEM_LINKING_CORD, SPECIES_ALAKAZAM}},
```

Comment:
Personally I would, if this affects every Pokémon (which is not the case in the mainline games btw.), just adapt the `EVO_TRADE` s.t. it works with this item, instead of adding a new evolution method for every single Pokémon.

Reply:
> which is not the case in the mainline games btw
It's official for every trade evolution in Legends Arceus that didn't have an item. When SwSh came out, Pokémon that evolve via magnetic fields were given the additional evolution method of using a Thunder Stone on them. At that time, Probopass wasn't in any Gen 8 games, but we thought to extrapolate that and we gave it that method too. In Legends Arceus, we were proven right when they gave it that Evolution Method while also still being able to evolve via Magnetic Field.
Based in all that, and due to the fact that in PLA you can still evolve those Pokémon by trading, it's fair to extrapolate Pokémon that aren't in PLA that have the `EVO_TRADE` method to also be able to use the Linking Cord.

Regarding adapting `EVO_TRADE` to also accept the Linking Cord, I think it's much more user friendly to let them remove the methods from the table than having to go into the code to remove that functionality of it. We didn't do that for Feebas' evolution method after all.